mantle/platform/qemu.go

```diff
@@ -363,6 +364,9 @@ func (builder *QemuBuilder) usermodeNetworkingAssignPorts() error {
 	if builder.Hostname != "" {
 		netdev += fmt.Sprintf(",hostname=%s", builder.Hostname)
 	}
+	if builder.RestrictNetworking {
+		netdev += ",restrict=on"
```

Comment:
Should this be `restrict=yes`? That's what I found in the [QEMU docs](https://wiki.qemu.org/Documentation/Networking#User_Networking_.28SLIRP.29).

Reply:
The man page says:
```
$ man qemu-kvm | grep restrict | head -1
           restrict=on|off
$
```

I also did test this locally as part of https://github.com/coreos/coreos-assembler/pull/1514

Reply:
Ah yeah, I'm seeing that locally as well. Those docs must be out of date.